composer.json

```diff
@@ -16,9 +16,9 @@
     "require": {
         "php": ">=5.5",
         "jms/serializer": "~1.5",
-        "symfony/routing": "~2.8|~3.0",
-        "symfony/yaml": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/routing": "~2.8|~3.0|~4.0",
```

Comment:
Do we really need to have 2-4 here? Can't we remove the `2.8`?

Reply:
We can.

Reply:
I'll leave it up to you :) Looks good

Reply:
Done.